server/planning/events/events_template.py

```diff
@@ -0,0 +1,248 @@
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+"""Superdesk Files"""
+
+import logging
+from flask import current_app as app
+from eve.utils import config, ParsedRequest
+from superdesk import Resource, get_resource_service
+from superdesk.services import BaseService
+from superdesk.metadata.item import metadata_schema
+from superdesk.errors import SuperdeskApiError
+from superdesk.utils import ListCursor
+
+logger = logging.getLogger(__name__)
+
+
+class EventsTemplateResource(Resource):
+    """
+    Resource for events template
+    """
+
+    endpoint_name = 'events_template'
+    resource_methods = ['GET', 'POST']
+    item_methods = ['GET', 'DELETE', 'PATCH', 'PUT']
+    privileges = {
+        'GET': 'planning_event_management',
+        'POST': 'planning_event_management',
+        'DELETE': 'planning_event_management',
+        'PATCH': 'planning_event_management',
+        'PUT': 'planning_event_management'
+    }
+    schema = {
+        'template_name': {
+            'type': 'string',
+            'required': True,
+            'empty': False,
+            'unique': True
+        },
+        'based_on_event': Resource.rel(
+            'events',
+            type=metadata_schema[config.ID_FIELD]['type'],
+            embeddable=False,
+            required=True
+        ),
+        # events related fields
```

Comment:
I think it's better to put this all into some dict so we don't mix template metadata with its data